server/internal/adapter/integration/item.go

```diff
@@ -103,26 +103,14 @@ func (s *Server) ItemsAsCSV(ctx context.Context, request ItemsAsCSVRequestObject
 	op := adapter.Operator(ctx)
 	uc := adapter.Usecases(ctx)
 
-	sp, err := uc.Schema.FindByModel(ctx, request.ModelId, op)
-	if err != nil {
-		return ItemsAsCSV400Response{}, err
-	}
-
-	p := fromPagination(request.Params.Page, request.Params.PerPage)
-	items, _, err := uc.Item.FindBySchema(ctx, sp.Schema().ID(), nil, p, op)
+	pr, err := uc.Item.ItemsAsCSV(ctx, request.ModelId, request.Params.Page, request.Params.PerPage, op)
```

Comment:
This looks good to me! 
Just a quick note: we have two functions that handle exporting items as CSV. You've already completed `ItemsAsCSV`, so please apply the same approach to the `ItemsWithProjectAsCSV` function. Once that's done, kindly remove the `csvFromItems` function from the adapter level.